src/index.html

```diff
@@ -6,9 +6,67 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home"><img src="images/logo.png"alt="moyo-logo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+            class="nav__link is-active">
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung"
+            class="nav__link">
+            Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones"
+            class="nav__link">
+            Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+            class="nav__link"
+            data-qa="hover">
+            Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Reply:
And here is the case when there are more then 2 attributes, so every attribute should start from new line.

Also pay attention that closing bracket has same indentation as open

Please fix everywhere